api-references/payments/prepaid-recharge.json

```diff
@@ -0,0 +1,1210 @@
+{
+    "info": {
```

Comment:
Currently the base URL is using a default `example.com`. Can we add a host, scheme etc similar to [this](https://github.com/SetuHQ/docs/blob/main/api-references/payments/bbps.json#L2-L10)?

<img width="1144" alt="Screenshot 2024-10-23 at 10 26 24" src="https://github.com/user-attachments/assets/239ca3be-33f5-4e27-b77c-a650557815d0">